eventmesh-meta/eventmesh-meta-zookeeper/src/main/java/org/apache/eventmesh/meta/zookeeper/service/ZookeeperMetaService.java

```diff
@@ -239,44 +206,49 @@ public List<EventMeshDataInfo> findAllEventMeshInfo() throws MetaException {
 
             String serviceName = entry.getKey();
             String clusterName = entry.getValue().getEventMeshClusterName();
-            try {
-                String servicePath = formatServicePath(clusterName, serviceName);
 
-                List<String> instances = zkClient.getChildren()
-                    .forPath(servicePath);
+            findEventMeshInfo("findAllEventMeshInfo", clusterName, serviceName, eventMeshDataInfoList);
+        }
+        return eventMeshDataInfoList;
+    }
 
-                if (CollectionUtils.isEmpty(instances)) {
-                    continue;
-                }
+    private void findEventMeshInfo(String tipTitle, String clusterName,
```

Comment:
感谢大佬们指点
Thank you for your guidance. My consideration is that the tipTitle here has two values, "findEventMeshInfoByCluster" and "findAllEventMeshInfo", whose call links are executed at different times. If there is an error, I am concerned that it may differ from the previous log printing rules